Jenkinsfile

```diff
@@ -25,7 +25,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 
-// @Library(value="pipeline-lib@my_pr_branch") _
+@Library(value="pipeline-lib@corci-768") _
```

Comment:
What if you have both uncommented but in an `if`/`else` type of conditional?

Reply:
According to the Jenkins documentation, the @Library statement is processed before the groovy code is run so it can only accept string constants available at that time.  The documentation shows using job parameters to control the library, but that will not help with triggered builds.

The dynamic syntax also will not allow replacing the default loaded library.  It does not error out the job, it just writes out a message in the log that the original library specification will be used.